test/k8s_metrics_tests/metric_data/selector.conf

```diff
@@ -0,0 +1,20 @@
+# This file is for documentation purposes only.
```

Comment:
why should this be a `conf` file? it is for documentation, can it be in markdown for better readability?
it is using a splunk conf file stanza. i think it can be confusing? 

so why do we need this documentation?

Reply:
We have name inconsistency across metric and metric-aggr plugin. So, I added this file for reference only so when we fix those inconsistency we don't get confused.
Markdown seems better for readability, I will update it

Reply:
let's be clear on what this doc is for and what "A=B" means in this doc.